test/mock-objects/mock-payment-timelines.js

```diff
@@ -0,0 +1,23 @@
+const moment = require('moment')

```

Comment:
Why have we created this module?

We already have `test/mock-objects/mock-schedule-periods.js` which already has these values with the exception of outstanding.

Furthermore, the values it has relate to a top-up payment request which means they actually relate to a potential input to the system. These are conjured out of thin air. If we had proper integration tests in this file where we pass in a top-up, reduction, recovery payment request etc this would almost certainly fail. Can you see what the purpose of the file `test/mock-objects/mock-schedule-periods.js` is doing?

I also assume this outstanding value is being sent out by this service for downstream? We need to add it to the async.

Reply:
We also need these created for a recovery scenario.

This looks like top-up and you can see in `test/mock-objects/mock-schedule-periods.js` it's created for top-up, therefore we need to create 1 for recovery and refactor accordingly